ajax/libs/angular-ui-codemirror/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-ui-codemirror",
+  "filename": "ui-codemirror.min.js",
+  "version": "0.3.0",
+  "description": "This directive allows you to add CodeMirror to your textarea elements.",
+  "author": "https://github.com/angular-ui/ui-codemirror/contributors",
+  "license": "MIT",
+  "homepage": "https://angular-ui.github.com",
+  "keywords": [
+    "CodeMirror",
+    "textarea",
+    "directive"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/angular-ui/ui-codemirror.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/angular-ui/ui-codemirror.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "**/*"
```

Comment:
@lcd78706 
The `fileMap` is wrong, you need to see the file structure of the version branch like **v0.3.0**, so the `basePath` should be `""` and the `files` should be `ui-codemirror*`.